util.go

```diff
@@ -1 +1,9 @@
 package jasper
+
+import "bytes"
+
+type writeCloser struct {
```

Comment:
This has already been defined in `command_test.go` so we should get rid of the duplicate bytes.Buffer wrapping.